CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ CHANGELOG
 ------
 
 **ENHANCEMENTS**
+- Add the config option `DefaultUserHome` under `HeadNode` to allow users to move the default user's home directory to `/local/home` instead of `/home` (default)
```

Comment:
* "config option" => "configuration parameter"
* Add full stop at the end of the sentence.